src/Assert.php

```diff
@@ -731,6 +836,9 @@ public static function alpha($value, $message = '')
         }
     }
 
+    /**
+     * @psalm-assert numeric $value
+     */
     public static function digits($value, $message = '')
```

Comment:
For all the other `ctype` based assertions, we can add `@psalm-assert string $value`. All ctype functions will return false if a non string value is passed into it.

Reply:
Never mind this suggestion as it is not true, `ctype` functions will also work by passing an int, which will be converted to its ASCII value.